src/main/java/de/terrestris/shogun/model/User.java

```diff
@@ -359,10 +358,11 @@ public void setActive(Boolean active) {
 	/**
 	 * @return the groups
 	 */
-	@ManyToMany(mappedBy = "users", fetch = FetchType.EAGER)
+	@ManyToMany(mappedBy = "users", fetch = FetchType.LAZY)
 	@JsonIgnore
 	@Fetch(FetchMode.SUBSELECT)
 	@JsonSerialize(using=LeanBaseModelSetSerializer.class)
+    @org.springframework.cache.annotation.Cacheable
```

Comment:
Indent.

Reply:
solved in https://github.com/terrestris/shogun-legacy/pull/70/commits/dfacf681b632a11b274e324d06efa5102ed97afc